config/chime_science_run_gpu_no_output.yaml

```diff
@@ -203,6 +203,14 @@ updatable_config:
           -1.1771713330451292e-32,
           3.948826407949732e-35,
         ],]
+  second_stage_rfi:
+    kotekan_update_endpoint: json
+    thresholds:
+      - threshold: 5.0
+        fraction: 0.025
+      - threshold: 1.4
+        fraction: 0.4
+    enable_rfi_zero: false
```

Comment:
This feels like a duplicate of `rfi_zeroing_toggle`.  I would consider either using that, or just using the metadata flag (see the other comment on this).

Reply:
We've renamed it so they have the same name, but I want them to be separately switchable. Rick has rewritten the coco endpoint such that one endpoint will call both (for the solar noon script)